netflow9/packet.go

```diff
@@ -290,10 +291,14 @@ func (tr TemplateRecord) register(s session.Session) {
 	s.AddTemplate(tr)
 }
 
-func (tr TemplateRecord) ID() uint16 {
+func (tr TemplateRecord) TID() uint16 {
 	return tr.TemplateID
 }
 
+func (tr TemplateRecord) OID() uint32 {
```

Comment:
The `netflow9.TemplateRecord` struct has to satisfy the <a href="https://github.com/scalingdata/netflow/blob/74385e0bf5e499da3cb55f3c78b82d596aec15b0/session/session.go#L7">session.Template interface</a>, so it has to use the `OID()` method defined by `session`. `SourceID` and `ObservationDomainID` are basically the same concept with different names due to the different protocol versions.